kinder/pkg/cluster/manager/actions/kubeadm-config.go

```diff
@@ -84,6 +84,13 @@ func KubeadmConfig(c *status.Cluster, kubeadmConfigVersion string, copyCertsMode
 		controlPlaneEndpoint = controlPlaneEndpointIPv6
 	}
 
+	featureGateName := ""
+	featureGateValue := ""
+	if featureGate != "" {
+		featureGateName = strings.Split(featureGate, "=")[0]
+		featureGateValue = strings.Split(featureGate, "=")[1]
```

Comment:
this line can panic if the user does not pass `=`.
better do the assignments like this:

```
if len(featureGate) > 0 {
	split := strings.Split(featureGate, "=")
	if len(split) != 2 {
		return errors.Errorf("feature gate must be formatted as 'key=value'")
	}
	featureGateName = split[0]
	featureGateValue = split[1]
}
```

Reply:
Done. Thanks!